qtodotxt/lib/tasklib.py

```diff
@@ -2,10 +2,27 @@
 import re
 
 from PyQt5 import QtCore
-
+from enum import Enum
 from qtodotxt.lib.task_htmlizer import TaskHtmlizer
 
 
+class recursiveModes(Enum):
+    init = 0     # Initial recursive mode
```

Comment:
> should be called RecursiveMode not RecursiveModeS
`class recursiveModes(Enum):
    completitionDate = 0    # Original due date mode: Task recurs from original due date
    originalDueDate = 1    # Completion date mode: Task recurs from completion date`
should list all possible modes. Why not calling it recursiveModes?
The mode itself is set in 
`class recursion:
    mode = None
    increment = None
    interval = None`
and the used variable which only holds THE current mode is called `mode`.